### PR TITLE
Implemented product class

### PR DESCRIPTION
**Pull request by Spicadox**
Implemented product class and the crud endpoints for issue #8. Had to opt for static methods(to save time) since class methods made the product model act funky and would overwrite current saves.

### CODE REVIEW

File: db_populateScript.py [hunk 1]
```diff
@@ -2,13 +2,11 @@
...
 from json import load
 from mongoengine import connect
-from models.Item import Item
 from models.Product import Product
-
 from models.User import User
 
 ## Replace it with you DB if you need to
-MONGODB_URL = "mongodb+srv://capstone:Capstone123@wesourcecluster01.ctf3x.mongodb.net/WesourceDatabase?retryWrites=true&w=majority"  # pylint: disable=line-too-long
+MONGODB_URL = "mongodb+srv://test2:123@cluster0.fujai.mongodb.net/test"  # pylint: disable=line-too-long
```
Comment (kaynoedah) [inline db_populateScript.py]: This change should be reverted.

File: index.py [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
 from util.decorators.errorHandler import exception_handler
+from product import Product, Products
 
+# Have to import models to register in the document registry
+import models.User  # pylint: disable=unused-import
+import models.Product  # pylint: disable=unused-import
 
 # Database URL
-MONGODB_URL = "mongodb+srv://capstone:Capstone123@wesourcecluster01.ctf3x.mongodb.net/WesourceDatabase?retryWrites=true&w=majority"  # pylint: disable=line-too-long
+MONGODB_URL = "mongodb+srv://test2:123@cluster0.fujai.mongodb.net/test"  # pylint: disable=line-too-long
```
Comment (kaynoedah) [inline index.py]: Revert this as well if this is your personal db.

**Review by kaynoedah — DISMISSED** [review]
LGTM

**Review by marajput123 — APPROVED** [review]
Look good to me.

**Pull request merged**